### DISCUSSION TITLE
NvChad + Julia-vim + autocomplete

### DISCUSSION THREAD

**Discussion by ivanuricardo** [NvChad/NvChad#2350] [Q&A] [answered]
Hello! 

I am a bit new to NvChad and was struggling a bit with the autocompletion while writing Julia. So Julia supports latex/unicode characters and I find myself trying to write "\mu" + <tab> but if there is an autocomplete suggestion, I can not write the latex character. This is because the autocompletion suggestions prevent me from pressing <tab> to create the \mu character. Are there any suggestions for getting around this? Thanks.

**Comment by siduck** [comment, marked as answer]
hi @ivanuricardo idk about julia but you dont need an external plugin for it i think, cant you just enable its lsp server in your lspconfig file? 

https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#julials

**Reply by ivanuricardo** [reply]
Ah no I have done this, but I think what might be better is to edit the key mappings of cmp. I've noticed I can edit the mappings in lua/plugins/configs and edit the cmp.lua file but I don't think this is the best practice correct? I have to edit the mappings in lua/custom/mappings.lua. 

Would I then just copy over the cmp.lua text into mappings.lua?
